qml/InitDSMonitorDialog.qml

```diff
@@ -0,0 +1,425 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import QtQuick 2.6
+import QtQuick.Layouts 1.3
+import QtQuick.Dialogs 1.2
+import QtQuick.Controls 2.15
+import Theme 1.0
+
+Dialog {
+    id: initDSMonitorDialog
+    modal: false
+    title: "Initialize Discovery Server Monitor"
+    standardButtons: Dialog.Apply | Dialog.Ok | Dialog.Cancel | Dialog.RestoreDefaults
+    implicitWidth: 600
+
+    x: (parent.width - width) / 2
+    y: (parent.height - height) / 2
+
+    onAccepted: {
+        initDiscoveryServer(true)
+    }
+
+    onApplied: {
+        initDiscoveryServer(false)
+        initDSMonitorDialog.open()
+    }
+
+    onReset: {
+        discoveryServerGuid.text = "44.53.00.5f.45.50.52.4f.53.49.4d.41"
+        discoveryServerLocatorsModel.clear()
+        discoveryServerLocatorsModel.append({"transportProtocolIdx": 0, "ip": "", "port": -1})
+    }
+
+    Component.onCompleted: {
+        standardButton(Dialog.Apply).text = qsTrId("Apply and Continue")
```

Comment:
Not sure this is useful.
I think that init one monitor in a discovery server is a very specific operation that it will not be repeated sequently very often (and if it does is just one click ahead).
AND, with this option it leads you to think that you can add locators to the already initialized monitor.

Reply:
Removed Appy button.